server.js

```diff
@@ -16,12 +21,172 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+
+const ThoughtsSchema = new mongoose.Schema({
+	//in this object specify every single thing about this property(name)
+	name: {
+		//adding parameters
+		// most impotant one is type
+		type: String,
+		default: "unknown",
+		// will require forces to provide this value
+		// required: true,
+		// new name will have to be different than all others in the database/it should be unique
+		// unique: true,
+		//to delete the whitespace
+		trim: true
+		//enum is handy for final project which is all the allowed values
+		// here in enum, only these "five" will be allowd in our database. nothing else rather than we write here is allowed here
+		// enum: ["Matilda", "Poya", "Petra", "Hanna", "Daniel"]
+	},
+	//can be sth similar to happy thought
+	message: {
+		type: String,
+		minlength: 5,
+		maxlength: 140,
+		required: true,
+		// trim removes unneccessary white spaces from the begining and from the end of our whole message
+		// by default is false and we here change it to true
+		trim: true
+	},
+	// score will only have 2 properties
+	hearts: {
+		type: Number,
+		//initil value if none other is specified
+		default: 0
+	},
+	//createdAt should have Date
+	createdAt: {
+		type: Date,
+		// new Date() will execute once - when we start the server
+		// default: new Date()
+		// onClick = { someFunction()}
+		// IIFE/immediatly function expression. //in the moment that you declare the function it is called/executed right away
+		// (() => new Date())()
+		// or it can be like this: (function functionName () {new Date()})()
+		default: () => new Date()
+	}
+});
+
+const Thoughts = mongoose.model("Thoughts", ThoughtsSchema);
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+	res.send({ Message: "Hello, Welcome to our Happy Thoughts!", data: listEndpoints(app) });
```

Comment:
In the information about the endpoints, It would be helpful to include information about what parameters the api/db takes and what is required for the POST/PATCH